app/src/main/java/com/kodeco/android/coordplot/MainActivity.kt

```diff
@@ -46,10 +97,36 @@ fun PlotSurfacePreview() {
 
 @Composable
 fun Map(xPercent: Float, yPercent: Float, modifier: Modifier = Modifier) {
-    // TODO fill out the square map here.
-    //  Create a Box composable with a size of 300.dp
-    //  and then create an inner Box composable
-    //  with a clip shape of CircleShape.
+    val density = LocalDensity.current.density
+    var boxSize by remember { mutableStateOf(Size(0, 0)) }
```

Comment:
Based on my other comment, we should be able to remove these variables altogether.